test/Conversion/ImportAIGER/basic.aag

```diff
@@ -0,0 +1,210 @@
+// RUN: circt-translate --import-aiger %s --split-input-file | FileCheck %s
+
+// Test basic AND gate
+// CHECK-LABEL: hw.module @aiger_top
+// CHECK-SAME:    (in %[[INPUT0:.+]] : i1, in %[[INPUT1:.+]] : i1, out [[OUTPUT0:.+]] : i1) {
+// CHECK-NEXT:    %[[AND:.+]] = aig.and_inv %[[INPUT1]], %[[INPUT0]] : i1
+// CHECK-NEXT:    hw.output %[[AND]] : i1
+// CHECK-NEXT: }
+aag 3 2 0 1 1
+2
+4
+6
+6 4 2
+
+// -----
+// Test invert
+// CHECK-LABEL: hw.module @aiger_top
+// CHECK-SAME:    (in %[[INPUT0:.+]] : i1, in %[[INPUT1:.+]] : i1, out [[OUTPUT0:.+]] : i1) {
+// CHECK-NEXT:    %[[AND:.+]] = aig.and_inv not %[[INPUT1]], not %[[INPUT0]] : i1
+// CHECK-NEXT:    hw.output %[[AND]] : i1
+// CHECK-NEXT: }
+aag 3 2 0 1 1
+2
+4
+6
+6 5 3
+
+// -----
+// Test multiple outputs
+// CHECK-LABEL: hw.module @aiger_top
+// CHECK-SAME:    (in %[[INPUT0:.+]] : i1, in %[[INPUT1:.+]] : i1, out [[OUTPUT0:.+]] : i1, out [[OUTPUT1:.+]] : i1) {
+// CHECK-NEXT:    %[[VAL0:.+]] = aig.and_inv %[[INPUT1]], %[[INPUT0]] : i1
+// CHECK-NEXT:    %[[VAL1:.+]] = aig.and_inv %[[VAL0]], %[[INPUT1]] : i1
+// CHECK-NEXT:    hw.output %[[VAL0]], %[[VAL1]] : i1, i1
+// CHECK-NEXT:  }
+aag 4 2 0 2 2
+2
+4
+6
+8
+6 4 2
+8 6 4
+
+// -----
+// Test latch
+// CHECK-LABEL:  hw.module @aiger_top
+// CHECK-SAME:     (in %[[INPUT0:.+]] : i1, out [[OUTPUT0:.+]] : i1, in %[[CLOCK:.+]] : !seq.clock) {
+// CHECK-NEXT:    %[[REG:.+]] = seq.compreg %[[AND:.+]], %[[CLOCK]] : i1
+// CHECK-NEXT:    %[[AND]] = aig.and_inv %[[REG]], %[[INPUT0]] : i1
+// CHECK-NEXT:    hw.output %[[REG]] : i1
+// CHECK-NEXT:  }
+aag 3 1 1 1 1
+2
+4 6
+4
+6 4 2
+
+// -----
+// Test with comments
+// CHECK-LABEL: hw.module @aiger_top
+// CHECK-SAME:    (in %[[INPUT0:.+]] : i1, in %[[INPUT1:.+]] : i1, out [[OUTPUT0:.+]] : i1) {
+// CHECK-NEXT:    %[[VAL:.+]] = aig.and_inv %[[INPUT1]], %[[INPUT0]] : i1
+// CHECK-NEXT:    hw.output %[[VAL]] : i1
+// CHECK-NEXT:  }
+aag 3 2 0 1 1
+2
+4
+6
+6 4 2
+c
+This is a comment line
+
+// -----
+// Test symbol table
+// CHECK-LABEL: hw.module @aiger_top
+// CHECK-SAME:    (in %[[IN1:.+]] : i1, in %[[IN2:.+]] : i1, out out : i1, in %[[CLOCK:.+]] : !seq.clock) {
+// CHECK-NEXT:    %[[LATCH:.+]] = seq.compreg %[[AND:.+]], %[[CLOCK]] : i1
+// CHECK-NEXT:    %[[AND]] = aig.and_inv %[[IN2]], %[[IN1]] : i1
+// CHECK-NEXT:    hw.output %[[AND]] : i1
+// CHECK-NEXT:  }
+aag 3 2 1 1 1
+2
+4
+8 6
+6
+6 4 2
+i0 in1
+i1 in2
+l0 latch
+o0 out
```

Comment:
Are these names preserved in the IR? If they are, it might make sense to explicitly match for them.